klippy/mcu.py

```diff
@@ -790,13 +790,11 @@ def is_fileoutput(self):
         return self._printer.get_start_args().get('debugoutput') is not None
     def is_shutdown(self):
         return self._is_shutdown
-    def flush_moves(self, print_time):
+    def flush_moves(self, step_gen_time, print_time):
         if self._steppersync is None:
             return
-        clock = self.print_time_to_clock(print_time)
-        if clock < 0:
-            return
-        ret = self._ffi_lib.steppersync_flush(self._steppersync, clock)
+        ret = self._ffi_lib.steppersync_flush(self._steppersync,
```

Comment:
```suggestion
        clock = self.print_time_to_clock(print_time)
        if clock < 0:
           return
        ret = self._ffi_lib.steppersync_flush(self._steppersync,
```

Not really understanding the code this may not be anything but could this be the root of some of the MCU errors that have been reported? Seems like the original check was probably there for a reason.